src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -235,14 +227,190 @@ object NguhBlocks {
+    val GILDED_CALCITE_STAIRS = RegisterStairs(GILDED_CALCITE)
+
+    val GILDED_POLISHED_CALCITE = Register(
+        "gilded_polished_calcite",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_POLISHED_CALCITE_SLAB = RegisterVariant(GILDED_POLISHED_CALCITE, "slab", ::SlabBlock)
+    val GILDED_POLISHED_CALCITE_STAIRS = RegisterStairs(GILDED_POLISHED_CALCITE)
+    val GILDED_POLISHED_CALCITE_WALL = RegisterVariant(GILDED_POLISHED_CALCITE, "wall", ::WallBlock)
+
+    val GILDED_CHISELED_CALCITE = Register(
+        "gilded_chiseled_calcite",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_CALCITE_BRICKS = Register(
+        "gilded_calcite_bricks",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val GILDED_CALCITE_BRICK_SLAB = RegisterVariant(GILDED_CALCITE_BRICKS, "slab", ::SlabBlock)
+    val GILDED_CALCITE_BRICK_STAIRS = RegisterStairs(GILDED_CALCITE_BRICKS)
+    val GILDED_CALCITE_BRICK_WALL = RegisterVariant(GILDED_CALCITE_BRICKS, "wall", ::WallBlock)
+
+    val GILDED_CHISELED_CALCITE_BRICKS = Register(
+        "gilded_chiseled_calcite_bricks",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    // =========================================================================
     // Block entities.
+    // =========================================================================
     val LOCKED_DOOR_BLOCK_ENTITY = RegisterEntity(
         "lockable_door",
         FabricBlockEntityTypeBuilder
             .create(::LockedDoorBlockEntity, LOCKED_DOOR)
             .build()
     )
 
+    // =========================================================================
+    // Block families
+    // =========================================================================
+    val CINNABAR_FAMILY = BlockFamilies.register(NguhBlocks.CINNABAR)
+        .polished(NguhBlocks.POLISHED_CINNABAR)
+        .slab(NguhBlocks.CINNABAR_SLAB)
+        .stairs(NguhBlocks.CINNABAR_STAIRS)
+        .build()
+    val POLISHED_CINNABAR_FAMILY = BlockFamilies.register(NguhBlocks.POLISHED_CINNABAR)
+        .polished(NguhBlocks.CINNABAR_BRICKS)
+        .slab(NguhBlocks.POLISHED_CINNABAR_SLAB)
+        .stairs(NguhBlocks.POLISHED_CINNABAR_STAIRS)
+        .wall(NguhBlocks.POLISHED_CINNABAR_WALL)
+        .build()
+    val CINNABAR_BRICK_FAMILY = BlockFamilies.register(CINNABAR_BRICKS)
+        .slab(NguhBlocks.CINNABAR_BRICK_SLAB)
+        .stairs(NguhBlocks.CINNABAR_BRICK_STAIRS)
+        .wall(NguhBlocks.CINNABAR_BRICK_WALL)
+        .build()
+    val CALCITE_FAMILY = BlockFamilies.register(Blocks.CALCITE)
+        .polished(NguhBlocks.POLISHED_CALCITE)
+        .build()
```

Comment:
```suggestion
```
We can’t just create a family for an existing block since that will mess w/ vanilla blocks; that’s the whole reason why we can’t just add stairs/slabs for regular calcite.